### DISCUSSION TITLE
Lunar Data Standards | Planetary SDI

### DISCUSSION THREAD

**Discussion by giscus** [DOI-USGS/planetary-sdi#8] [General]
# Lunar Data Standards | Planetary SDI

The standards enumerated herein are intended to facilitate data interoperability. How is the lunar SDI defining data interoperability? A user of data made available by a data provider, via a data portal that has adopted the Lunar SDI standards can reasonably assume that different data sets will:
Be made available in the same, GIS ready formats. Data conforming to these standards are interoperable at a technical level by the tools and technologies used to analyze and view the data.

http://localhost:1313/moon/standards/

**Comment by jlaura** [comment]
Test for title.
